### PR TITLE
output: use detector names as ntuple name

### PR DESCRIPTION
**Pull request by ManuelHu**
see #153 for discussion

* not yet an option (but an option would be easy to add)
* no solution for the flat output mode yet

### CODE REVIEW

**Comment by ManuelHu** [comment]
@gipert now the option is added, should be ready

File: src/RMGManager.cc [hunk 1]
```diff
@@ -376,18 +376,24 @@ void RMGManager::DefineCommands() {
   fOutputMessenger->DeclareProperty("NtuplePerDetector", fOutputNtuplePerDetector)
       .SetGuidance("Create a ntuple for each sensitive detector to store hits. Otherwise, store "
                    "all hits of one detector type in one ntuple.")
-      .SetParameterName("tree_per_det", false)
+      .SetParameterName("nt_per_det", false)
+      .SetStates(G4State_PreInit, G4State_Idle);
+
+  fOutputMessenger->DeclareProperty("NtupleUseVolumeName", fOutputNtupleUseVolumeName)
+      .SetGuidance("Use the sensitive volume name to name output ntuples.")
+      .SetGuidance("note: this only works if `NtuplePerDetector` is set to false.")
```
Comment (gipert) [inline src/RMGManager.cc:384]: set to true?

Comment (ManuelHu) [reply src/RMGManager.cc]: yes

**Comment by gipert** [comment]
Looks good to me.

**Pull request merged**